engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/DisplayTypeNode.java

```diff
@@ -0,0 +1,16 @@
+package org.enso.interpreter.node.expression.builtin.meta;
+
+import com.oracle.truffle.api.CompilerDirectives.CompilationFinal;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.node.expression.builtin.text.util.TypeToDisplayTextNode;
+import org.enso.interpreter.runtime.data.text.Text;
+
+@BuiltinMethod(type = "Meta", name = "display_type", description = "Pretty prints a type.")
```

Comment:
Please rename to something like `type_to_display_text`. `display_type` suggests that the resulting text is going to be actually displayed.